lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -134,6 +134,19 @@ def #{configuration[:column]}=(position)
             after_update :update_positions_if_necessary, unless: -> { ActiveRecord::Acts::List.skip_cb? }
 
             scope :in_list, lambda { where("#{table_name}.#{configuration[:column]} IS NOT NULL") }
+
+            def self.insert_after_record_with_position(records, targeted_record_position)
```

Comment:
What about changing this to `move_records_to_position(records, position)` to make it a little easier to understand the name/intent. This would mean removing the `+ 1` from this method, and leaving the caller to pass in the desired target position themselves.

Reply:
That seems like a reasonable argument. 👍